contracts/oracle/src/contract/alarms.rs

```diff
@@ -311,23 +280,23 @@ mod test {
         for x in 0..=5 {
             let delta = x * 10;
 
-            let addr = Addr::unchecked(format!("addr1_{}", delta));
+            let receiver = Addr::unchecked(format!("receiver1_{}", delta));
 
             MarketAlarms::try_add_price_alarm::<Base>(
                 &mut storage,
-                addr,
+                receiver,
                 AlarmDTO::new(
                     price::total_of(Coin::<Atom>::new(1)).is(Coin::<Base>::new(10 + delta)),
                     Some(price::total_of(Coin::<Atom>::new(1)).is(Coin::<Base>::new(30 + delta))),
                 ),
             )
```

Comment:
with the changed result type, please craft more tests and tailor them to cover more specific cases, e.g. various combinations of above/below alarms and receivers. The asserts should check the addresses, not just their length.

Reply:
that is **very important to be tested thoroughly** because a bug here would have huge implication to the system as a whole

Reply:
it's much better to have more shorter tests than a few longer